Userland/Libraries/LibCore/System.h

```diff
@@ -43,6 +43,7 @@ ErrorOr<void> mount(int source_fd, StringView target, StringView fs_type, int fl
 ErrorOr<void> umount(StringView mount_point);
 ErrorOr<long> ptrace(int request, pid_t tid, void* address, void* data);
 ErrorOr<void> disown(pid_t pid);
+ErrorOr<void> profiling_enable(pid_t, uint64_t);
```

Comment:
The second parameter wants to be named here, since it's not obvious. First one being nameless is fine though.

Reply:
Also we tend to prefer `u64` from AK/Types.h rather than `uint64_t` from `<stdint.h>` for new code.